IPFS-scholar.md

```diff
@@ -0,0 +1,61 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+*IPFS-scholar/hfs-ipfs-scholar*
```

Comment:
Hey there! Would you be able to include your individual Filecoin slack handle here as well so we can contact you on Slack if needed?

Reply:
I have updated the information in the new submission. Please check.